docker/README.md

```diff
@@ -0,0 +1,24 @@
+## Building and using Docker development images.
+
+These Dockerfiles will create a DronecodeSDK development container image based on whichever flavour of Ubuntu or Fedora you prefer.
+
+To create an image :-
+
+```bash
+cd /whereever/DronecodeSDK/docker
+docker build -t dronecode -f Dockerfile-Ubuntu-18.04 .
```

Comment:
If you tag it to `dronecode/dronecode-sdk-ubuntu-18.04`, you don't actually need to build this yourself and can just pull it from dockerhub:
https://hub.docker.com/r/dronecode/dronecode-sdk-ubuntu-18.04/

I suggest also to use this tag in the run_docker.sh script. It makes it easier, I think.

Reply:
Are you saying that the docker build is not needed? is ci building and pushing to docker hub?

Reply:
I'm currently doing it manually using: https://github.com/Dronecode/DronecodeSDK/blob/develop/docker/build_and_push_docker_images.sh.

Reply:
ah ok, so do you want to leave the reference to building it locally (with the tag changed) in the README in case someone wants to build locally? or remove altogether?